examples/step-57/doc/results.dox

```diff
@@ -0,0 +1,426 @@
+    <th>1.39986e-14 </th>
+    <th>5         </th>
+    <th>2.18864e-14 </th>
+    <th>5         </th>
+    <th>3.38787e-14 </th>
+    <th>5         </th>
+  </tr>
+
+  <tr>
+    <th>5          </th>
+    <th>1.34601e-5</th>
+    <th>4          </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+
+  <tr>
+    <th>6         </th>
+    <th>2.5235e-8    </th>
+    <th>5    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+
+  <tr>
+    <th>7         </th>
+    <th>1.38899e-12    </th>
+    <th>4    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+
+  <tr>
+    <th>8           </th>
+    <th>4.68224e-15 </th>
+    <th>4           </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+    <th>&nbsp;      </th>
+  </tr>
+</table>
+
+The following is the generated grids during solving. For this case of Re=400,
+the initial guess is obtianed by solving Stokes on $8 \times 8$ mesh, and the
+mesh is refined adaptively. Between meshs, the solution from coarse is
+transfered to fine mesh as an initial guess.
+
+<TABLE WIDTH="60%" ALIGN="center">
+  <tr>
+    <td ALIGN="center">
+      <img src="step-57_Re400_Mesh0.png" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="step-57_Re400_Mesh1.png" alt="">
+    </td>
+  </tr>
+
+  <tr>
+    <td ALIGN="center">
+      <img src="step-57_Re400_Mesh2.png" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="step-57_Re400_Mesh3.png" alt="">
+    </td>
+  </tr>
+
+  <tr>
+    <td ALIGN="center">
+      <img src="step-57_Re400_Mesh4.png" alt="">
+    </td>
+  </tr>
+</table>
+
+This picture is the graphical result of lid-drive cavity with Re=400.
+<img src="step-57_Re400_Streamline.png" alt="">
+
+Then the solution by Newton's iteration is compared with a reference solution
+from High-Re Solutions for Incompressible Flow Using the Navier-Stokes
+Equations and a Multigrid Method, U. Ghia, K.N.Ghia and C.T.Shin.
+
+<img src="step-57-compare-Re400.svg" alt="">
+ 
+<h3> Test case 1: Re=10000 </h3>
+
+Newton's iteration requires a good initial guess. However, the nonlinear term
+gets dominant when Reynold number is large, so that the solution to Stokes
+equations may be far away from the exact solution. If the Stokes solution acts
+as the initial guess, the convergence will be lost. The following picture
+shows that the residual can not get lower as iteration going on.
```

Comment:
... shows that the nonlinear iteration gets stuck and the residual no longer decreases with further iterations.

A side note: Is this because the system gets into some pseudo-transient state that does not decay (or decay only very slowly)?